packages/twenty-website/src/content/user-guide/functions/workflows.mdx

```diff
@@ -0,0 +1,151 @@
+
+### Updated Record
+
+This action makes changes to an existing record of a selected object. To use this action, select the object and select the specific record you want to update. Then, pick the field(s) to modify and enter the new values. Updated field data can be used in later steps of the workflow.
+
+### Delete Record
+
+This action removes a record from a selected object. To use this action, select the object and record you want to delete. 
+
+Note: Data from the deleted record remains available for use in later workflow steps.
+
+### Search records
+
+This action finds records within a selected object using filter conditions. To use this action, select the object you wish to search through and set the criteria to narrow down the search results.
+
+### Send Emails
+
+This action allows you to send an email from your workflow. To use this action:
+
+1. Add an email account in **Settings > Accounts**
+2. Enter the recipient’s address, subject, and message body to compose your email
+
+You can also reference variables from outputs of previous steps to further customize an action. Soon, you will have the capability to view attachments and format email content. 
+
+### Code 
+
+This action runs custom Javascript within your workflow. You can use this action to handle logic that goes beyond built-in actions such as transforming data, performing calculations, or integrating with external APIs. 
+
+Any variables you create and return here can be reused in later steps of the workflow. You can also test your code directly in this step and view the output to confirm it works as expected.
+
+Note: You can manage keys and access your API Playground [here](https://app.twenty.com/settings) or navigate to Settings > API & Webhooks (under Developers).
+
+### Form 
+
+This action allows you to prompt a form during the workflow to collect input from the user. The data collected can be used in later steps. To add this action, configure the input fields by selecting the type, label, and placeholder for each. 
+
+### HTTP Request
+
+This action allows you to send a request to an external API as part of your workflow; it can be used to integrate with external services, send or receive data, or trigger third-party actions. To use this action:
+
+1. Paste the endpoint you want to call
+2. Select an HTTP method from `GET`, `POST`, `PUT`, `PATCH`, or `DELETE`
+3. Enter any required header inputs and their values 
+4. For `POST`, `PUT`, or `PATCH`  include the body input to send with the request
+5. Paste a sample response body to preview its structure and reference keys in later steps
+
+## Executing Workflows
+
+Workflows can be executed in three ways: **Activate**, **Run**, and **Test**. Each serves a different purpose depending on where you are in the workflow setup.
+
+### Activate 
+
+To activate your workflow, click **Activate** at the top to run your workflow in the background. Each activation creates a new version and moves the workflow out of draft mode.
```

Comment:
Activating a workflow is like publishing it to production. Triggers will start listening to events (scheduling, record is created/updated/deleted, etc.)